web/impact/impact/permissions/v1_api_permissions.py

```diff
@@ -91,9 +90,6 @@ def has_object_permission(self, request, view, office_hour):
                 office_hour.mentor == request.user and not is_reserved)
 
     def get_message(self, request):
-        if request.method == 'POST':
-            return CREATE_PERMISSION_DENIED_DETAIL
-        elif request.method == 'PATCH':
-            return EDIT_PERMISSION_DENIED_DETAIL
-        else:
-            return DEFAULT_PERMISSION_DENIED_DETAIL
+        dict = {'POST': CREATE_PERMISSION_DENIED_DETAIL,
```

Comment:
overwriting the name `dict` is allowed in python, but it's bad practice. How about `detail_messages`?